tests/gold_tests/autest-site/trafficserver.test.ext

```diff
@@ -125,7 +136,6 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True,
     #########################################################
     # create subdirectories that need to exist (but are empty)
     # log directory has to be created with correct permissions
-    p.Setup.MakeDir(log_dir)  # log directory has to be created
     p.Setup.Chown(log_dir, "nobody", "nobody", ignore=True)
     # covers ubuntu's unprivileged group
     p.Setup.Chown(log_dir, "nobody", "nogroup", ignore=True)
```

Comment:
I think we can replace these Chown with one `traffic_layout verify --fix` command

Reply:
Interesting idea, but I don't think that's an option for the test. The `verify` sub command requires it to be run with root privileges:

```
ERROR: To fix permission issues, root privilege is required.
Please run with sudo.

```

Reply:
@dragon512 ?

Reply:
I think we need to tweak something more